src/software/backend/output/radio/mrf/messages.h

```diff
@@ -205,4 +205,6 @@ namespace MRF
         "Second dongle on this channel+PAN";
     static constexpr const char* TRANSMIT_QUEUE_FULL_MESSAGE = "Transmit Queue Full";
     static constexpr const char* RECEIVE_QUEUE_FULL_MESSAGE  = "Receive Queue Full";
+    static constexpr const char* PACKET_ABOVE_MAX_SIZE =
+        "Attempted to send packet over radio greater then the maximum size";
```

Comment:
Nit: Is there a way to format to also print out the maximum size here?

Reply:
We _could_ make it a shared variable, but I really want to limit leakage for anything associated with this old system outside the folders for it....... @MathewMacDougall @jonathanlew - your thoughts?

Reply:
Is the max radio transfer size not already defined somewhere in the `radio` folder? I agree I don't want things leaking (so it's easy to remove) but that seems like something specific enough to radio that it should already be defined in there somewhere.